.travis.yml

```diff
@@ -5,7 +5,7 @@ jobs:
   include:
     - if: (fork IN (true)) OR (type IN (pull_request))
       os: osx
-      osx_image: xcode6.4
+      osx_image: xcode9.4
```

Comment:
This can be submitted in an independent PR, which we should merge promptly.

xref: https://github.com/conda-forge/conda-forge.github.io/issues/641